tox.ini

```diff
@@ -8,12 +8,27 @@ commands = python -m pytest --cov=beamlime --cov-report html:coverage_html/pytho
 
 [testenv:deps]
 description = Update dependencies by running pip-compile-multi
-basepython = python3.9
 skip_install = true
 deps = pip-compile-multi
 changedir = requirements
 commands = pip-compile-multi -d .
 
+[testenv:config-build]
+description = Build new default configuration yaml file
```

Comment:
Can you extend this description, it is not clear to me as an outsider when and how this should be used/run.